docs/datatypes/mesh/index.md

```diff
@@ -2576,6 +2576,16 @@ init();</script></body>
 </html>" width="100%" height="500px" style="border:none;"></iframe
 
 
+To display a colored point cloud, store the corresponding colors in the `.tensor` of a chunk `Document` with the name tag `'point_cloud_colors'`. The colors have to be of shape (n_samples, 3) or (n_samples, 4).
+
+```python
+n_samples = 1000
+colors = np.random.rand(n_samples, 3)
+doc = Document(
+    uri='mesh_man.glb', chunks=[Document(tensor=colors, name='point_cloud_colors')]
+).load_uri_to_point_cloud_tensor(samples=n_samples)
+```
```

Comment:
Maybe we can switch this code snippet around to first load the point coordinate like always, and then add the color chunk in a second step